src/doc/style-guide/src/expressions.md

```diff
@@ -663,7 +663,8 @@ never use a block (unless the block is empty).
 
 If the right-hand side consists of multiple statements, or has line comments,
 or the start of the line does not fit on the same line as the left-hand side,
-use a block.
+use a block. Do not flatten a right-hand side block containing a single macro call
+because its expanded form could contain a trailing semicolon.
```

Comment:
Another option was to leave this block of text as-is, but to then add some sentence explicitly for tools, e.g. "tools must never remove existing braces because..."

I opted against that given our recent push to remove tool-targeted language from the guide

Reply:
Yeah, I think this language is better.